docs/tutorials/tutorial-0.rst

```diff
@@ -1,175 +1,65 @@
-Tutorial 0 - Sandbox
-====================
+Tutorial 0 - Getting Started
```

Comment:
Also, I didn't want to merge 0 and 1 together, because then the one tutorial would be too big.